c2corg_api/views/user_mailinglists.py

```diff
@@ -0,0 +1,48 @@
+from c2corg_api import DBSession
+from c2corg_api.models.user import User
+from c2corg_api.models.mailinglist import Mailinglist
+from c2corg_api.views import cors_policy, restricted_json_view, \
+    restricted_view
+from c2corg_common.attributes import mailinglists
+from cornice.resource import resource
+from colander import MappingSchema, SchemaNode, String, Boolean, Sequence, \
+    OneOf, required
+from sqlalchemy.orm import joinedload, load_only
+
+
+@resource(path='/users/mailinglists', cors_policy=cors_policy)
+class UserMailinglistsRest(object):
+
+    def __init__(self, request):
+        self.request = request
+
+    @restricted_view()
+    def get(self):
+        """Get the mailinglists subscriptions of the authenticated user.
+
+        Request:
+            `GET` `/users/mailinglists`
+
+        Example response:
+
+            {'avalanche': False, 'lawinen': True, 'valanghe': False, ...}
+
+        """
+        user_id = self.request.authenticated_userid
+        listnames = DBSession.query(Mailinglist.listname). \
+            options(joinedload(Mailinglist.user).load_only(User.email)). \
+            filter(User.id == user_id). \
+            all()
+
+        return {ml: ml in mailinglists for ml in listnames}
+
+    @restricted_json_view(schema=MailinglistSchema())
```

Comment:
With the update of Cornice you will have to specify the schema validator explicitly:

```
from cornice.validators import colander_body_validator
...
    @restricted_json_view(schema=MailinglistSchema()), validators=[colander_body_validator])

```